### ISSUE TITLE
Autocomplete from dot

### ISSUE THREAD

**Issue by paarthenon** [shikijs/twoslash#63]
Is it possible to autocomplete from the `.` (dot) of the property accessor? 

In short, I'd like to write something equivalent to this simplified example

```ts twoslash
// @noErrors
const obj = {
    alpha() { }
    beta() { }
    gamma() { }
};

obj.
//  ^|
```

and have `alpha`, `beta`, and `gamma` populated in the list.

I'm trying to demonstrate in my tutorial that an object I generate is limited to certain properties and if this were working it would be an excellent way to do that. As things are, this is what I see:

![image](https://user-images.githubusercontent.com/10353928/124816410-81922a00-df36-11eb-95ce-4dfd0bcdeecf.png)

Thanks for taking a look.

**Comment by orta** [comment]
Agree, this is a bug all the way at [ts-twoslash](https://github.com/microsoft/TypeScript-website/blob/v2/packages/ts-twoslasher/) level - I think this is just a case I didn't account for (as I _did_ want the prefix in all my usecases)
